### ISSUE TITLE
Stored properties cannot be marked potentially unavailable with '@available'

### ISSUE THREAD

**Issue by wf1992** [pichillilorenzo/flutter_inappwebview#1310]
<img width="1254" alt="image" src="https://user-images.githubusercontent.com/12962946/189830305-00725723-cc6b-4d7e-bab9-3b90de0e384d.png">

Stored properties cannot be marked potentially unavailable with '@available'

@available(iOS 14.0, *)
    lazy var contentWorld: WKContentWorld = WKContentWorld.page
    
    can you fix this bug in Xcode 14?

**Comment by aiqianqianli** [comment]
I also encountered this problem

**Comment by iBog** [comment]
appeared after releasing xcode 14

**Comment by lam-nv** [comment]
My team is also facing the same problem

**Comment by mdddj** [comment]
same question

**Comment by lam-nv** [comment]
refer to MR #1238

**Comment by jdumont0201** [comment]
Same here since the Xcode 14 update, targeting iOS 11


**Pubspec** 
`flutter_inappwebview: ^5.4.3+7`  
**Xcode** 
```

Xcode 14.0
Build version 14A309
```
**Flutter Doctor**
```
[✓] Flutter (Channel stable, 3.3.1, on macOS 12.6 21G115 darwin-arm, locale en-GB)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 14.0)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2021.2)
[✓] VS Code (version 1.71.0)
[✓] Connected device (3 available)
[✓] HTTP Host Availability
```

**Comment by mirkancal** [comment]
> My current fix is:
> 
> Update version in Podfile platform :ios, '14.0'
> 
> Reset pods cd ios && rm -rf Pods/ Podfile.lock && pod install --repo-update

Maybe "your current fix" has a source?
https://stackoverflow.com/a/73699790/9779791

**Comment by jdumont0201** [comment]
#1216 has another fix as forked in https://github.com/CodeEagle/flutter_inappwebview

the fix
https://github.com/pichillilorenzo/flutter_inappwebview/compare/master...CodeEagle:flutter_inappwebview:master

**Comment by EvGeniyLell** [comment]
hot fix could be like that 
```
    private var _contentWorld: NSObject?
    @available(iOS 14.0, *)
    var contentWorld: WKContentWorld {
        get {
            return (_contentWorld as? WKContentWorld) ?? WKContentWorld.page
        }
        set {
            _contentWorld = newValue
        }
    }
```
instead of old
```
    @available(iOS 14.0, *)
    lazy var contentWorld: WKContentWorld = WKContentWorld.page
```

**Comment by Dorian0796** [comment]
> hot fix could be like that
> 
> ```
>     private var _contentWorld: NSObject?
>     @available(iOS 14.0, *)
>     var contentWorld: WKContentWorld {
>         get {
>             return (_contentWorld as? WKContentWorld) ?? WKContentWorld.page
>         }
>         set {
>             _contentWorld = newValue
>         }
>     }
> ```
> 
> instead of old
> 
> ```
>     @available(iOS 14.0, *)
>     lazy var contentWorld: WKContentWorld = WKContentWorld.page
> ```
> 
> **please, can you quickly add this to master ?**

This works, please add this to master?

**Comment by wf1992** [comment]
this bug was fixed in new code version
